elephant/signal_proc.py

```diff
@@ -0,0 +1,130 @@
+'''
+signal_proc module
```

Comment:
I think we should avoid using abbreviations* in module or public function names, so I would make this `signal_processing`.

*widely-used acronyms (e.g. fft, sta) I think are ok, however.